back/app/views/application/_header.mjml

```diff
@@ -1,5 +1,4 @@
-
-<% unless header_logo_only? %>
+<% unless preheader.empty? %>
```

Comment:
We always want the preheader, even in the 3 core emails, just as we do for all in email_campaigns.

This is now equivalent to the emails_campaigns [_header.mjml](https://github.com/CitizenLabDotCo/citizenlab/blob/master/back/engines/free/email_campaigns/app/views/email_campaigns/application/_header.mjml) implementation